### PR TITLE
Increased cloud build timeout to 75 seconds instead of 50

### PR DESCRIPTION
**Pull request by limonspb**
Seems like current cloud build timeout sometimes is no enough, because i keep sing this (sometimes):
image

This PR increases timeout to 75 seconds from 50 seconds.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by limonspb** [comment]
Caught one more timeout error on build.
Looks like the cloud is busy sometimes, its not he build itself taking too long.
https://build.betaflight.com/api/builds/0c0159e30cf717c7ffd64ba6ae3eb0a7/log
image

But since it's happening time to time, i think it's worth increasing timeout in Configurator.

**Comment by McGiverGim** [comment]
Other build systems, like OpenWrt, show the position in the queue. In this way the user knows more or less the time to wait. I think it will be good to show this info in some way, or at least have two timeouts: one to advance in the queue and other for the build itself.

**Comment by limonspb** [comment]
> Other build systems, like OpenWrt, show the position in the queue. In this way the user knows more or less the time to wait. I think it will be good to show this info in some way, or at least have two timeouts: one to advance in the queue and other for the build itself.

I think adding this new functionality at RC stage could be dangerous. Now this is a simple solution to mitigate most of the timeout issues, unless the queue is a little long.

**Comment by nerdCopter** [comment]
https://build.betaflight.com/healthcheck shows the queue.
@blckmn controls the timeout on API side IIRC.

**Review by ctzsnooze — APPROVED** [review]

**Comment by haslinghuis** [comment]
60 seconds :)

File: src/js/tabs/firmware_flasher.js
Comment (haslinghuis) [inline src/js/tabs/firmware_flasher.js:40]: Lets do 60 seconds :)

Comment (haslinghuis) [reply src/js/tabs/firmware_flasher.js:40]: Also don't get timeouts lately anymore.

Comment (limonspb) [reply src/js/tabs/firmware_flasher.js:40]: i think its already 60 now? I got timeout error just a few days ago

**Comment by blckmn** [comment]
This is not the right way to go about it. It's by design that it times out and says retry. This gives you the option of exiting and doing something else. Making it 75 just prolongs this.

If the build server is busy you'll get a timeout. Retrying won't resubmit the job and double up because the API checks the queue for your build key, and just starts the wait process again.

This is just aesthetics.

Be better to wait indefinitely and give you a button to cancel the wait if you're really worried about the aesthetics.

**Review by blckmn — CHANGES_REQUESTED** [review]
As per comment. There are better options.

**Pull request closed without merging**
